Makefile

```diff
@@ -16,97 +16,103 @@ METADATA_DEFAULTS_FILE=${MAKEFILE_PATH}/pkg/config/defaults/aemm-metadata-defaul
 ENCODED_METADATA_DEFAULTS=$(shell cat ${METADATA_DEFAULTS_FILE} | base64 | tr -d \\n)
 DEFAULT_VALUES_VAR=github.com/aws/amazon-ec2-metadata-mock/pkg/config/defaults.encodedDefaultValues
 
+version:
```

Comment:
while we're here, can you modify line 1 of this Makefile w/ a `?=` so that we can mutate the version. 

This is needed for the homebrew tap build. 

`VERSION ?= $(shell git describe --tags --always --dirty)`